tests/test_fyle/test_tasks.py

```
@@ -9,13 +9,14 @@
 from rest_framework.exceptions import ValidationError
 
 from apps.fyle.actions import mark_expenses_as_skipped
-from apps.fyle.models import Expense, ExpenseGroup, ExpenseGroupSettings
+from apps.fyle.models import Expense, ExpenseGroup, ExpenseGroupSettings, ExpenseFilter
```

Ruff (0.8.2)

`apps.fyle.models.ExpenseFilter` imported but unused

Remove unused import: `apps.fyle.models.ExpenseFilter`

(F401)